test/DataMCSpectraComparison/histosSimplified.py

```diff
@@ -298,7 +307,8 @@ def ntuplify(process, fill_gen_info=False):
         if fill_gen_info:
             process.pathOur2016 *= obj * process.SimpleNtupler
             if Electrons:
-                process.SimpleNtuplerEmu = process.SimpleNtupler.clone(dimu_src = cms.InputTag('SimpleMuonsElectronsAllSigns'))
+                #process.SimpleNtuplerEmu = process.SimpleNtupler.clone(dimu_src = cms.InputTag('SimpleMuonsElectronsAllSigns'))
+                process.SimpleNtuplerEmu = process.SimpleNtupler.clone(dimu_src = cms.InputTag('Our2016ElectronsPlusElectronsMinus'))
```

Comment:
You should not change the `SimpleNtuplerEmu`, but instead create a new subprocess with the name `SimpleNtuplerDiEle` or so